Formula/sshuttle.rb

```diff
@@ -4,8 +4,8 @@ class Sshuttle < Formula
   desc "Proxy server that works as a poor man's VPN"
   homepage "https://github.com/sshuttle/sshuttle"
   url "https://github.com/sshuttle/sshuttle.git",
-      tag:      "v1.0.3",
-      revision: "c5dcc918db666dfd1b30afc72cc198abfb3b3aa9"
+      tag:      "v1.0.4",
+      revision: "abb48f199656f776be0f05601ff5746f36df3370"
   license "LGPL-2.0"
```

Comment:
```suggestion
  license "LGPL-2.1-or-later"
```

[license header:](https://github.com/sshuttle/sshuttle/blob/73eb3b6479d3605d557e4481a0c609d2f18abe4b/setup.py#L7-L10)

```
# sshuttle is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as
# published by the Free Software Foundation; either version 2.1 of
# the License, or (at your option) any later version.
```

However, the license file included in the repo is `LGPL-2.0`. Might be worth opening an issue upstream.

Reply:
This doesn't clear it up: https://github.com/sshuttle/sshuttle/issues/186

Reply:
They did update the repo file now

Reply:
> They did update the repo file now

Can you clarify what you mean? I don't think anything has changed since I last looked.

Reply:
But the [`LICENSE`](https://github.com/sshuttle/sshuttle/blob/master/LICENSE) file still has the `LGPL-2.0` license

Reply:
Somehow I didn't realize that issue was 3 years old.

Reply:
I guess we should make a new issue.

Reply:
👍 Am I right in saying the only part that's confusing is the `LICENSE` file? If we feel good that the rest is `LGPL-2.1` I might just open a PR to update the `LICENSE` file. It also looks like we're more likely to get a timely response that way.

Reply:
https://github.com/sshuttle/sshuttle/pull/511

Reply:
The PR was merged, so the next `sshuttle` release should have the new `LGPL-2.1` license in the `LICENSE` file.

Reply:
Thanks for all that work. If I understand correctly, `"LGPL-2.1-or-later"` is the way to go for this release too.